### PR TITLE
Adds components

### PR DESCRIPTION
**Pull request by ronitonit**

### CODE REVIEW

File: src/compontents/Sub-header/Subheader.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+                    <button>Login with Facebook</button>
+                </div>
+                    
+            </div>
+
+        )
+    }
+}
+
+export default Subheader;
```
Comment (ronitonit) [inline src/compontents/Sub-header/Subheader.js]: a blank line below this is ne necessary

File: src/compontents/Sub-header/Subheader.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+                <div>
+                    <p>Because there is no single </p>
+                    <p>BEST song. </p>
+                    <p>Just favourites</p>
+                    <p>Find hidden musical gems buried within your fren’s profile.</p>
+                </div>
+                <div>
+                    <button>Login with Facebook</button>
+                </div>
+
```
Comment (ronitonit) [inline src/compontents/Sub-header/Subheader.js]: bhai lets try to have a nice indentation. 
maybe there is a plugin to do this.

File: package.json [hunk 3]
```diff
@@ -6,7 +6,7 @@
     "react": "^16.8.1",
     "react-dom": "^16.8.1",
     "react-router-dom": "^4.3.1",
-    "react-scripts": "2.0"
+    "react-scripts": "^2.1.5"
```
Comment (ronitonit) [inline package.json]: is this the latest ?

File: src/compontents/Sub-header/Subheader.js (hunk 2 above)
Comment (niroz11) [reply src/compontents/Sub-header/Subheader.js]: I agree with that. We can set up linting to get proper indentation. will work on it

**Pull request merged**